params/protocol_params.go

```diff
@@ -179,10 +180,12 @@ var (
 	FullerMapContext = big.NewInt(0)           // Fork = 0 meaning initial ontology
 	TuringMapContext = big.NewInt(math.MaxInt) // maxed out for now
 	LovelaMapContext = big.NewInt(math.MaxInt) // maxed out for now
+
 	// Named ontologies
-	FullerOntology   = []int{3, 3, 3} // named after Buckminster Fuller
-	TuringOntology   = []int{4, 4, 4} // named after Alan Turing
-	LovelaceOntology = []int{5, 5, 5} // named after Ada Lovelace
+	// This defines the number of allowed regions (r) and zones (z) in the hierarchy (r,z)
+	FullerOntology   = []int{3, 3} // named after Buckminster Fuller
```

Comment:
Am I understanding these correctly? An ontology of `[3,3]` would correspond to 3 regions & 3 zones indexed 1-3? So `region-0` would be invalid?

Reply:
so this is r and z.  We should leave these as 1 indexed because they are physical value reps and just make sure we use them correctly when comparing to zero indexed values.